src/pages/billing/Project.js

```diff
@@ -320,15 +322,15 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
   const reloadBillingProjectUsers = _.flow(
     reportErrorAndRethrow('Error loading billing project users list'),
     Utils.withBusyState(setUpdating)
-  )(() => Ajax(signal).Billing.project(billingProject.projectName).listUsers()
+  )(() => Ajax(signal).Billing.listProjectUsers(billingProject.projectName)
     .then(collectUserRoles)
     .then(setProjectUsers)
   )
 
   const removeUserFromBillingProject = _.flow(
     reportErrorAndRethrow('Error removing member from billing project'),
     Utils.withBusyState(setUpdating)
-  )(Ajax().Billing.project(billingProject.projectName).removeUser)
+  )(_.partial(Ajax().Billing.removeProjectUser, [billingProject.projectName]))
```

Comment:
So these are our first instances of `_.partial` in the codebase; we normally declare any function likely to be used like this with `_.curry` instead. That said, this feels to me like a reason to revert the structure in the ajax file to how it was before, which was done with the intention of how it was going to be used in mind.

Reply:
If using the `project` wrapper was a means of achieving partial application then that's another reason against it. As a side note, I don't like this practice of defining curried functions. If it's a normal function then I can choose what to do with it. If I want to, I can curry it or partially apply arguments or just call it without a ton of parentheses. Let the caller decide.

Reply:
So, having now looked at the docs, I understand what this is doing but I found it less readable, personally, then the previous code. Especially in the case of the props below. However, in this case I _personally_ don't have strong opinions on it, but just wanted to throw my two cents in.

Reply:
Follow-up - I didn't realise lodash's `curry` allowed you to pass more than one argument (ie `f(a)(b)(c)` can be called `f(a, b, c)` when `f` is `_.curry`'ed.